src/Job/ToolRunner.ts

```diff
@@ -52,27 +53,49 @@ export class ToolRunner {
   tag = this.constructor.name;  // logging tag
 
   // This variable is undefined while a prcess is not running
-  private child: cp.ChildProcessWithoutNullStreams|undefined = undefined;
+  private child: cp.ChildProcess|undefined = undefined;
```

Comment:
(Q) Why do you change to `ChildProcess`?

Reply:
I looked into the code and found how I can use previous `ChildProcessWithoutNullStreams`.
Let me use `ChildProcessWithoutNullStreams` again. Thanks.